config/yugabyte/regression_pipelines/miscellaneous/postgres/MG2_multi_statement_transaction.yaml

```diff
@@ -119,16 +119,16 @@ microbenchmark:
                               -   util: RandomNoWithDecimalPoints
                                   params: [ 1000000, 10000000, 2 ]
                               -   util: PrimaryIntGen
-                                  params: [ 1, 1000000 ]
+                                  params: [ 1, 10000000000 ]
```

Comment:
Rakshit the changes done are incorrect. The issue cause of which this microbenchmark was insufficient unique primary keys while inserting in the table audit_table_1 for which I increased the range to 10000000000.
The benchmark has got 3 tables users_1 , accounts_1 and audit_table_1 each with 1M rows inserted in load phase. In execute phase there are multiple queries running part of single workload:

- INSERT on audit_table_1 << Issue for which the range was increased to 10000000000
- UPDATE on accounts_1 << For rest of the queries we can update/select on the same 1M rows we inserted in load phase we dont need to increase the range for this.
- SELECT on accounts_1
- UPDATE on users_1

The problem is framework should allocated the range at query level and not at workload level.